### PR TITLE
Updated README.md to add online version

### PR DESCRIPTION
**Pull request by sabertazimi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 * [Day5 - Updating - Diff](blog/update-contd.md)
 * [Day6 - Updating - Real DOM Update](blog/update-dom.md)
 
+You can also read [Gitbook Online Version](https://dragonforker.github.io/learn-react-source-code)
```
Comment (cyan33) [inline README.md]: How about: 😄 

```
[Read it Online](https://dragonforker.github.io/learn-react-source-code)
```

Comment (sabertaz) [reply README.md]: It's better

**Comment by sabertazimi** [comment]
Done.

**Comment by cyan33** [comment]
Thanks @sabertazimi 👍

**Pull request merged**